config.yml

```diff
@@ -3,3 +3,6 @@ logging:
 
   loggers:
     com.heroku.kafka.demo: DEBUG
+
+kafka:
```

Comment:
This is how Dropwizard does the mapping of the env var to `KafkaConfig.java`. Advantage of doing it this way is we can validate the var at boot time. Previously it was reading the env var at request time.